src/api/app/views/webui/request/_actions_details.html.haml

```diff
@@ -23,6 +23,9 @@
                                                                         diff_to_superseded: diff_to_superseded_id),
                                                       class: "dropdown-item #{action_item.id == active_action.id ? 'active' : ''}",
                                                       'aria-current': 'true')
+    - if User.session
```

Comment:
Ideally yes, but there is no real good place at the moment from my POV.
Placing it on a new line after the action description is something that looks really weird.

Actually I'd better refactor the `Select Action` dropdown to display the initial part of the title of the selected one. With that, having the `Action seen` check right after the dropdown would be perfectly related. But this is for another PR probably. What do you think?

Reply:
Yeah, I agree. I'll start by moving it for now. As soon as some better options comes up, we can always change it.

Btw yes it can be related to the card you linked above, and also to this one https://trello.com/c/1Pg8ZhVO